.github/workflows/deploy_devel.yaml

```diff
@@ -3,7 +3,8 @@ concurrency: pages_deployment
 
 on:
   push:
-    branches: [ master ]
+    branches: [ mute ]
+  workflow_dispatch:
```

Comment:
I added `workflow_dispatch` to development deployment on master, so you can drop this change after rebasing.